flink-table/flink-table-api-java-bridge/src/main/java/org/apache/flink/table/factories/DataGenTableSourceFactory.java

```diff
@@ -85,8 +60,17 @@ public String factoryIdentifier() {
     @Override
     public Set<ConfigOption<?>> optionalOptions() {
         Set<ConfigOption<?>> options = new HashSet<>();
-        options.add(ROWS_PER_SECOND);
-        options.add(NUMBER_OF_ROWS);
+        options.add(DataGenTableSourceFactoryOptions.ROWS_PER_SECOND);
```

Comment:
make the code more readable by calling this `DataGenOptions` similar to `KafkaOptions` etc.